src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -115,6 +113,13 @@ public void run() {
             throwIfCancelled();
             updateMetadata(module, outputFile);
             throwIfCancelled();
+            for (Converter converter : converters) {
+                Path outputPath = converter.convert(outputFile);
```

Comment:
You're right, I don't think it's different. Unless you care about the filename.

Either way, you can save a line.